web/src/__snapshots__/ErrorPane.test.tsx.snap

```diff
@@ -39,6 +39,45 @@ exports[`renders multiple lines of a crash log 1`] = `
 </section>
 `;
 
+exports[`renders multiple lines of a crash log 2`] = `
```

Comment:
isn't the snapshot above (`renders multiple lines of a crash log 1`) obsolete now?

Reply:
yessss, (kind of! there are two snapshots because a bad merge led to there being two identical tests with this name, so jest stored them with suffixes 1 and 2, which is why jest didn't delete the obsolete one)